src/d3a/d3a_core/simulation.py

```diff
@@ -444,20 +438,33 @@ def _handle_input(self, console, sleep: float = 0):
             if sleep == 0 or time.monotonic() - start >= sleep:
                 break
 
-    def _handle_paused(self, console):
-        if self.pause_after and self.time_since_start >= self.pause_after:
-            self.paused = True
-            self.pause_after = None
+    def _handle_paused(self, console, tick_start):
+        if console is not None:
+            self._handle_input(console)
+            if self.pause_after and self.time_since_start >= self.pause_after:
+                self.paused = True
+                self.pause_after = None
+
+        paused = False
         if self.paused:
+            if console:
+                log.critical("Simulation paused. Press 'p' to resume or resume from API.")
+            else:
+                self._update_and_send_results()
             start = time.monotonic()
```

Comment:
Since we are here, how about removing the monotonic clock in favor of the normal clock (time.time) ? Thanks!